packages/x-components/src/x-modules/url/components/url-handler.vue

```diff
@@ -4,10 +4,19 @@
   import { XOn } from '../../../components/decorators/bus.decorators';
```

Comment:
There are some issues that I need to investigate deeper.

1. Extra params are added immediately to the URL, even if it is the default value. I guess this is because of the order
2. Extra param change causes unstable history:
	* Search `lego`
	* Select a filter
	* Change the store to `Italy` -> Filters are then deselected
	* Hit back in the browser -> Filter is selected back, but store is kept in `Italy`
	* This happens because the filters are cleared in another batch of url updates.
3. Related tag selection is not undoable:
	* Search `playmobil`
	* Select any related tag
	* Hit back -> The tag is removed from the URL, but no new request is made.

Reply:
4. Selected filters component should exclude raw filters from being paint.